### PR TITLE
Chore: Setup sherif

### LINKED ISSUE #1495: [Feature]: Add sherif
### Description

we need tooling to align dep version across all our packages as the repo gros

### Explanation

_No response_

### PR DESCRIPTION
**Pull request by swalker326**
## Description

Ran Sherif and resolved all conflicts
Added script to run sherif

### DONE
~~Need to add to CI, I think I know where but kind of a weak spot for me~~

## Related Issue
Closes #1495 

## Motivation and Context
Packages in monorepo can get out of sync, this gives us a tool to sort out what is out of sync

## How Has This Been Tested?

image

### One test timed out and I couldn't get it to NOT time out

image

image

### CODE REVIEW

**Comment by bautistaaa** [comment]
what're your thought on sherif. seem worth it?

**Comment by swalker326** [comment]
> what're your thought on sherif. seem worth it?

Yea I think so, repos seems like it's at a good size for it.

File: apps/admin/src/app/dashboard/reports/[id]/_components/extra-actions.report.tsx
Comment (Hacksore) [inline apps/admin/src/app/dashboard/reports/[id]/_components/extra-actions.report.tsx:1]: not sure why this file was added but probably harmless

**Review by Hacksore — APPROVED** [review]

File: apps/admin/src/app/dashboard/reports/[id]/_components/extra-actions.report.tsx
Comment (swalker326) [reply apps/admin/src/app/dashboard/reports/[id]/_components/extra-actions.report.tsx:1]: I added the component as the build had a wall of warnings from million js

screenshot_2024-08-29_23-31-31

**Comment by swalker326** [comment]
Think this is good to go

**Pull request merged**